.github/workflows/ci.yml

```
@@ -125,11 +125,38 @@ jobs:
+      - name: Install Dependencies
+        working-directory: backend
+        run: npm install
+
+      - name: Build Application
+        working-directory: backend
+        run: npm run build
+        env:
+          NODE_ENV: staging
+
+      - name: Generate Version ID
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)